src/Backend/Core/Engine/Header.php

```diff
@@ -152,13 +153,17 @@ public function addCSS(
         Priority $priority = null
     ): void {
         $module = $module ?? $this->url->getModule();
+        $isExternalUrl = $this->get('fork.validator.url')->isExternalUrl($file);
+        $overwritePath = $overwritePath || $isExternalUrl; // external urls always overwrite the path
+        $minify = $minify && !$isExternalUrl;
```

Comment:
Am I correct if I say the files are always minified, even in debug-mode?
In my personal opinion this is a good thing, but I can think of some people (who aren't using gulp an write plain css instead of using sass) that can have some issues.

Reply:
No, the debug check is inside the minifier.

Reply:
👍